opbeat/contrib/django/management/commands/opbeat.py

```diff
@@ -0,0 +1,287 @@
+        888   888  888   888  888   888 888ooo888  .oP"888    888
+        888   888  888   888  888   888 888    .o d8(  888    888 .
+        `Y8bod8P'  888bod8P'  `Y8bod8P' `Y8bod8P' `Y888""8o   "888"
+                   888
+                  o888o
+
+"""
+
+CONFIG_EXAMPLE = """
+
+You can set it in your settings file:
+
+    OPBEAT = {
+        'ORGANIZATION_ID': '<YOUR-ORGANIZATION-ID>',
+        'APP_ID': '<YOUR-APP-ID>',
+        'SECRET_TOKEN': '<YOUR-SECRET-TOKEN>',
+    }
+
+or with environment variables:
+
+    $ export OPBEAT_ORGANIZATION_ID="<YOUR-ORGANIZATION-ID>"
+    $ export OPBEAT_APP_ID="<YOUR-APP-ID>"
+    $ export OPBEAT_SECRET_TOKEN="<YOUR-SECRET-TOKEN>"
+    $ python manage.py opbeat check
+
+"""
+
+
+class Command(BaseCommand):
+    option_list = BaseCommand.option_list + (
+        make_option('-o', '--organization-id', default=None,
+                    dest='organization_id',
+                    help='Specifies the organization ID.'),
+        make_option('-a', '--app-id', default=None,
+                    dest='app_id',
+                    help='Specifies the app ID.'),
+        make_option('-t', '--token', default=None,
+                    dest='secret_token',
+                    help='Specifies the secret token.'),
+    )
+    args = 'test check'
+
+    def handle(self, *args, **options):
+        if not args:
+            self.handle_command_not_found('No command specified.')
+        elif args[0] not in self.dispatch:
+            self.handle_command_not_found('No such command "%s".' % args[0])
+        else:
+            self.dispatch.get(
+                args[0],
+                self.handle_command_not_found
+            )(self, args[0], **options)
+
+    def handle_test(self, command, **options):
+        """Send a test error to Opbeat"""
+        self.stdout.write(LOGO, cyan)
+        config = get_client_config()
+        # can't be async for testing
+        config['async'] = False
+        for key in ('organization_id', 'app_id', 'secret_token'):
+            if options.get(key):
+                config[key] = options[key]
+        client_class = get_client_class()
+        client = client_class(**config)
+        client.error_logger = Logger(self.stderr)
+        client.logger = Logger(self.stderr)
+        client.state.logger = client.logger
+        client.state.error_logger = client.error_logger
+        self.stdout.write(
+            "Trying to send a test error to Opbeat using these settings:\n\n"
+            "ORGANIZATION_ID:\t%s\n"
+            "APP_ID:\t\t\t%s\n"
+            "SECRET_TOKEN:\t\t%s\n"
+            "SERVERS:\t\t%s\n\n" % (
+                client.organization_id,
+                client.app_id,
+                client.secret_token,
+                ', '.join(client.servers)
+            )
+        )
+
+        try:
+            raise OpbeatTestException('Hi there!')
+        except OpbeatTestException as e:
+            result = client.captureException()
```

Comment:
Yes, it does, but since it will not be in-app, it's going to be collapsed. But not much we can do about that :)